src/app/portlets/dot-edit-page/content/dot-edit-content.component.spec.ts

```diff
@@ -805,4 +804,31 @@ describe('DotEditContentComponent', () => {
             });
         });
     });
+
+    describe('Auto save', () => {
+        it('should call the save endpoint after a model change happens', () => {
+            const model: DotPageContainer[] = [
+                {
+                    identifier: '1',
+                    uuid: '2',
+                    contentletsId: ['3', '4']
+                }
+            ];
+
+            const newModel: DotPageContainer[] = [
+                {
+                    identifier: '2',
+                    uuid: '3',
+                    contentletsId: ['4', '5']
+                }
+            ];
+
+            spyOn(component, 'saveContent');
+            fixture.detectChanges();
+            dotEditContentHtmlService.pageModelChange.next(model);
+            fixture.detectChanges();
+            dotEditContentHtmlService.pageModelChange.next(newModel);
+            expect(component.saveContent).toHaveBeenCalled();
```

Comment:
we don't need check if the saveCotent method is called, we should check if the dotEditPageService,save method is called instead